application/models/get.php

```diff
@@ -105,12 +105,12 @@ public function categoryExists($id) {
         return true;
     }
 
-    public function getIdeasByCategory($category, $order, $type, $page){
+    public function getIdeasByCategory($category, $order, $type, $page, $status){
         $page = (int) $page;
     	$category = (int) $category;
         $max = $this->getSetting('max_results');
         $from = ($page - 1) * $max;
-    	$query = "SELECT * FROM ideas WHERE categoryid='$category' AND status !='new' ORDER BY ";
+    	$query = "SELECT * FROM ideas WHERE categoryid='$category' AND status = '$status' ORDER BY ";
```

Comment:
It seems the logic is flipped here. Why?

Reply:
This PR proposes a mechanism for filtering ideas by status. Here is the db query for fetching our data. `new` is still filtered out, but in the controller side. IMO, it's not the storage responsibility to assert that a status is never accessible.